include/wlr/types/wlr_seat.h

```diff
@@ -252,10 +252,25 @@ struct wlr_seat {
 		struct wl_signal request_set_cursor;
 
 		// wlr_seat_request_set_selection_event
+
+		// Called when an application _wants_ to set the selection (user copies some data).
+		// More specifically, the application wants to set a wlr_data_source object for the selection.
+		// This is what a compositor usually listens for.
```

Comment:
Maybe this is slightly better?

"Compositors should listen to this event and call wlr_seat_set_selection if they want to accept the client's request."